cylc/flow/xtrigger_mgr.py

```diff
@@ -87,11 +88,17 @@ class XtriggerManager(object):
 
     """
 
-    def __init__(self, suite: str, user: str, *,
```

Comment:
What the heck was the `*` doing here?

Reply:
Looks like a new syntax in Python 3. All arguments following the asterisk are called by keywords only. I'll reinstate as it was lost in my change.